frontend/components/bc/BcCurrencySelection.vue

```diff
@@ -2,10 +2,12 @@
 import type { Currency } from '~/types/currencies'
 
 defineProps<{
-  showCurrencyIcon: boolean
+  showCurrencyIcon: boolean,
 }>()
 
-const { currency, withLabel, setCurrency } = useCurrency()
+const {
+  currency, setCurrency, withLabel,
+} = useCurrency()
```

Comment:
fyi: should be like that, but atm could not get autofixed, as the rule: 
https://eslint.style/rules/js/object-property-newline#object-property-newline
lacks a proper destructuring logic
```suggestion
const {
  currency,
  setCurrency,
  withLabel,
} = useCurrency()
```